geops/openlayers-editor
# refactor(modify)!: split Modify control in multiple interactions

# How to

- Updated getFeatureAtPixel in ModifyControl
- Now includes selectFilter to filter selectable features

Testable on mapset:
- Log into https://editor.dev.mapset.ch/
- Switch to RBS or VBZ Corporate Design
- Draw a Picture and a few other features
- Switch to SBB Corporate Design
- The Picture should not be selectable and the cursor should be 'default' on hover
- Other feature should be selectable as always
- Switch back to RBS or VBZ
- The Picture should be selectable and the cursor should be 'pointer' on hover


# Others



- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [ ] IE11 tested.
- [x] The title is formatted as a [conventional-commit](https://www.conventionalcommits.org/) message.
- [x] The title contains `WIP:` if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?


IMPORTANT: Squash commits before or on merge to prevent every small commit being written into the change log. The Pull Request title will be written as message for the new commit containing the squashed commits and there fore needs to be in conventional-commit format
